commit 24825eb1d7d20c23b3ac771ff29ad5e0066aa9a9
Author: Martin Vician <martin.vician@nic.cz>
Date:   Mon May 2 18:41:44 2016 +0200

    fixed missing color

diff --git a/src/block.c b/src/block.c
index 7a72c0c..792a06e 100644
--- a/src/block.c
+++ b/src/block.c
@@ -163,7 +163,8 @@ block_update_plain_text(struct block *block, char *buf)
                 return;
 
 	linecpy(&lines, props->short_text, sizeof(props->short_text) - 1);
-	linecpy(&lines, props->color, sizeof(props->color) - 1);
+	if (*lines)
+		linecpy(&lines, props->color, sizeof(props->color) - 1);
 }
 
 static void
